app/assets/javascripts/components/ReactionDetails.js

```diff
@@ -181,10 +181,8 @@ export default class ReactionDetails extends Component {
       return false;
     } else {
       return (
-        <Col md={9}>
-          <div style={svgContainerStyle}>
-            <SVG key={reaction.svgPath} src={reaction.svgPath} className='reaction-details'/>
-          </div>
+        <Col md={12}>
```

Comment:
variable svgContainerStyle is still present at function 'reactionSVG' definition. But we don't use it anymore.
So plz remove it from function definition if it is not needed